vulcanai2/models/basenetwork.py

```diff
@@ -199,8 +238,8 @@ def get_output_shapes(self, input_size=None):
         :return: OrderedDict of shape of each layer in the network
         """
         if not input_size:
-            if self._input_network:
-                input_size = self._input_network._dimensions
+            if self._input_networks:
+                input_size = self._input_networks._dimensions
```

Comment:
remember that you have a list for `_input_networks` so just calling `_dimesions` will not work. you will need to dynamically calculate it considering you concatenate the outputs. Also, we should have multiple checks and warning for non-matching incoming shapes (e.g. ConvNet and DenseNet incoming) so the user can decide the method for combining the two